lib/vsc/mympirun/mpi/mpi.py

```diff
@@ -151,8 +151,37 @@ def which(cmd):
     return None
 
 
-class MPI(object):
+class RunFileLoopMPI(RunFile, RunLoop):
+    """
+    RunFile specific for MPI because intel doesn't feel like fixing their shit
+    """
+    def __init__(self, cmd, **kwargs):
+        self.options = kwargs.pop('options', None)
```

Comment:
add a docstring please

just pass in the value for `output` and `timeout`, rather than *all* options?

```python
self.output = kwargs.pop('output', None)
self.timeout = kwargs.pop('timeout', None)
```